kaska/tests/test_s2_obs.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+"""Test reading s2_observations data"""
+import datetime as dt
+import os
+import shutil
+import sys
+from pathlib import Path
+
+import numpy as np
+import pytest
+
+from ..inverters import get_emulator
+from ..s2_observations import Sentinel2Observations
+
+TEST_PATH = Path(os.path.dirname(__file__))
+
+
+def test_s2_data():
```

Comment:
I can't call this test a unit test.

Reply:
I'm not calling it a unit test! :upside_down_face: